src/main/java/htsjdk/samtools/Defaults.java

```diff
@@ -79,6 +79,14 @@
      */
     public static final String CUSTOM_READER_FACTORY;
 
+    /**
+     * Pathname to a local disk directory housing a cache of reference files.
+     * The pathname can be constructed using %nums and %s notation, consuming num characters of the MD5sum.
+     * For example /local/ref_cache/%2s/%2s/%s will create 2 nested subdirectories with the filenames in
+     * the deepest directory being the last 28 characters of the md5sum.
+     */
```

Comment:
I have updated the PR. Regarding unsetting the property after the tests try/catch blocks will not help because the REF_CACHE variable is a static constant and therefore it cannot be modified. I have addressed the issue by adding two new Gradle test tasks.